shared/utils/getCurrency.ts

```diff
@@ -8,6 +8,8 @@ export const getCurrencyFromManager = (
   const currencyResult =
     currency.type === Types.RequestLogic.CURRENCY.ETH
       ? currencyManager.getNativeCurrency(currency.type, currency.network!)
+      : currency.type === Types.RequestLogic.CURRENCY.ISO4217
+      ? currencyManager.fromSymbol(currency.value, currency.network)
       : currencyManager.fromAddress(currency.value, currency.network);
```

Comment:
TypeScript will complain using the proposed object mapping because `getNativeCurrency()` only works with native currencies like `CURRENCY.ETH` but the object mapping doesn't know that `currency.type` is a native currency. Resolved as No Problem :ok_person: